### PR TITLE
WIP: Rework on HELM

### PR DESCRIPTION
**Pull request by pozgo**

### CODE REVIEW

File: internal/cli/aux.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package cli
```
Comment (dfernandezm) [inline internal/cli/aux.go:1]: `cli_validation_utils` could be better suited name

File: internal/cli/aux_test.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import (
+	"log"
+	"os"
+	"testing"
+)
+
+func TestSetHelmChartsPathVar(t *testing.T) {
+	setHelmChartsPathVar("/test")
+	if os.Getenv("CHARTS_PATH") != "/test" {
```
Comment (dfernandezm) [inline internal/cli/aux_test.go:11]: You should use the Go Assertion util https://github.com/stretchr/testify that is used in other tests `assert.Equal(t, "/test", os.Getenv("CHARTS_PATH"))

File: internal/deploy/deploy.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ func applyDNSThroughSdk(newIP string) {
 // moneycolfrontend, moneycolserver, elasticsearch, traefik, traefik-dev, collections-api
 func Deploy(appName string, environment string, propertiesMap map[string]string, dryRun bool) {
 	helmSetParams := make(map[string]string)
-	if appName == "traefik" || appName == "traefik-dev" {
+	if appName == "traefik" || appName == "traefik-dev" || appName == "test" {
```
Comment (dfernandezm) [inline internal/deploy/deploy.go:61]: All this moneycol stuff I need to move away from here, could you please create issues for this and assign them to me?

**Review by dfernandezm — COMMENTED** [review]
just 2 minor things I spotted, I know this is all WIP

**Comment by davidfernandezm** [comment]
@pozgo I though the best way to rework this is avoid the need of having `helm` binary around and use the official Go client like here `https://github.com/mittwald/go-helm-client/blob/master/client.go.`. Much more portable.

**Pull request closed without merging**